src/types.rs

```diff
@@ -97,16 +104,28 @@ impl Configuration {
 
     /// POST Holo-hosted hApp URLs to resolver
     pub fn update_happ2host(&self) -> Result<()> {
+        const RETRIES: usize = 1;
+        const DELAY: u64 = 1000; // ms
+        const URL: &str = "https://resolver.holohost.net/update/addHost";
```

Comment:
there's something fishy about using CONST in function for me. CONST stays remains in memory for entire duration of program, while you need values declared here only in scope of this function block. wouldn't simple `let` be better? Especially that you do `let mut retries = RETRIES;` below anyway.